drv/riscv-plic-server/build.rs

```diff
@@ -41,15 +29,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         File::create(out.join("plic_config.rs")).unwrap()
     };
 
-    let plic_base: u32 = {
-        let mut path: String = env::var("HUBRIS_CHIP_DIR").unwrap();
```

Comment:
Are you planning to revert [this change](https://github.com/rivosinc/hubris/commit/3012e07b98df41ee159f2935d201786cd3dff193) for `HUBRIS_CHIP_DIR` now?

Reply:
Ah yes, I'll add that on into this patch series.